.circleci/config.yml

```diff
@@ -12,6 +12,8 @@ jobs:
     docker:
       - image: circleci/python:3.6.9
         environment:
+          ALLOWED_HOSTS: localhost
+          CONNECT_METHOD: http
```

Comment:
Truly, we don't need these, but I think we should have them match what's in docker-compose.yaml until we know better.